### PR TITLE
[@types/faker] adds missing typings for faker.unique method

### PR DESCRIPTION
**Pull request by pallymore**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/Marak/faker.js/blob/master/lib/unique.js#L24-L35
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## faker ([<kbd>unpkg</kbd>](https://unpkg.com/browse/faker@latest/))
was missing the following properties:
1. locales
1. localeFallback
1. definitions
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 870b77c838660d7a98e875a02baff8833396e035
</p>

**Comment by pallymore** [comment]
HI -  @bensw @basp @Kuniwak @mattbishop @testica @TastefulElk @damonpam  

When you get a chance, would you mind taking a look at the new typings here and see if they are ok?

Thanks!

File: types/faker/index.d.ts [hunk 1]
```diff
@@ -273,6 +273,12 @@ declare namespace Faker {
             vin(): string;
             color(): string;
         };
+
+        unique<T extends (...args: any) => any>(
+            method: T,
+            args?: Parameters<T>,
+            opts?: { maxTime?: number; maxRetires?: number },
```
Comment (TastefulElk) [inline types/faker/index.d.ts:280]: "Retries" is misspelled

**Review by TastefulElk — CHANGES_REQUESTED** [review]

**Comment by TastefulElk** [comment]
Requested to fix spelling error - other than that, awesome work and than you for contributing 👑

File: types/faker/index.d.ts (hunk 1 above)
Comment (pallymore) [reply types/faker/index.d.ts:280]: my bad! thanks!

**Comment by pallymore** [comment]
> Requested to fix spelling error - other than that, awesome work and than you for contributing 👑

fixed now 😅  thank you!

**Review by TastefulElk — APPROVED** [review]

**Comment by pallymore** [comment]
Ready to merge

**Pull request merged**